pyannotate_tools/annotations/tests/parse_test.py

```diff
@@ -123,6 +124,11 @@ def test_any_and_unknown(self):
         self.assert_type_comment('(Any) -> pyannotate_runtime.collect_types.UnknownType',
                                  ([any_arg()], AnyType()))
 
+    def test_no_return(self):
```

Comment:
Would it make sense to add a test case to `pyannotate_tools/fixes/tests/test_annotate_json.py` as well?

Reply:
I don't think it would add anything; to that phase of the program there's nothing special about NoReturn.